db.json

```diff
@@ -1,5 +1,190 @@
 {
-  "posts": [{ "id": 1, "title": "json-server", "author": "typicode" }],
-  "comments": [{ "id": 1, "body": "some comment", "postId": 1 }],
-  "profile": { "name": "typicode" }
-}
+  "attendees": [
+    {
+      "name": "Test Name",
+      "id": 1
+    },
+    {
+      "id": 2
+    },
+    {
+      "id": 3
+    },
+    {
+      "id": 4
+    },
+    {
+      "address": "addresse",
+      "id": 5
+    },
+    {
+      "state": {
+        "name": "",
+        "": "das"
+      },
+      "id": 6
+    },
+    {
+      "state": {
```

Comment:
Find die Idee genau wie Franz hammer! Mir ist aber aufgefallen, dass du quasi immer ein neues "state":{wirkliches Object} pair erstellst, anstatt die Objekte direkt in die Datenbank zu speichern. Glaube das gibt später Probleme.